src/prog_translator.h

```diff
@@ -62,7 +65,7 @@ class ProgTranslator {
   /// @brief translates solution from iface back into graph matches
   void FromProg();
 
-  const ProgTranslator::Context& ctx() const { return ctx_; }
+  const Context& ctx() const { return ctx_; }
```

Comment:
Although it is possible for users to depend on this code, it is considered part of our "internal" API.  And, off the top of my head I can't think of any code that a user could write that this would cause to not compile (i.e. this is not really an API change - although certainly is more ABI-breaking-ish).  And we not infrequently make changes to code that require users to recompile their code.  This change was *required* to eliminate some 3rd party dependency header leakage.  If somebody knows of users who are adversley affected by this change or knows of a better way to eliminate header leakage, than reverting would be no problem.  Otherwise, I think we would be shooting ourselves in the foot.